src/errors.rs

```diff
@@ -141,6 +141,12 @@ pub enum RustupError {
         target: TargetTriple,
         suggestion: Option<String>,
     },
+    #[error(
+        "unable to correctly link proxies\n\
+        help: this might be a bug in rustup, please open a new issue here:\n\
+        help: https://github.com/rust-lang/rustup/issues/new"
+    )]
+    InvalidProxyInstall,
```

Comment:
I'd avoid this wording and variant name since technically the proxy install completed without error -- it's just that the installed proxy does not actually work.

Reply:
@djc How about:

```suggestion
    BrokenProxies,
```

... and

 `"the proxies installed by rustup don't seems to work"`